### PR TITLE
Cleanup everything.

### PR DESCRIPTION
**Pull request by wolfd**
- Move map stuff into BootprintMapFragment
- Subfolders
- New network code

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -77,7 +77,9 @@ dependencies {
     compile 'com.google.android.gms:play-services:10.0.1'
     compile 'com.android.support:design:25.0.1'
 
-    // fast android networking
+    // retrofit
     compile 'com.squareup.retrofit2:retrofit:2.1.0'
     compile 'com.squareup.retrofit2:converter-jackson:2.1.0'
+    compile 'com.jakewharton.retrofit:retrofit2-rxjava2-adapter:1.0.0'
+    compile 'com.google.android.gms:play-services-maps:10.0.1'
```
Comment (sammyers) [inline app/build.gradle:86]: what is this

File: app/src/main/java/website/bloop/app/LocationModel.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package website.bloop.app;
+
+import android.location.Location;
+
+import io.reactivex.Observable;
+import io.reactivex.subjects.PublishSubject;
+
+public class LocationModel {
```
Comment (sammyers) [inline app/src/main/java/website/bloop/app/LocationModel.java:8]: Hmm interesting

File: app/src/main/java/website/bloop/app/activities/PlayLoginActivity.java [hunk 3]
```diff
@@ -103,27 +95,26 @@ public void onConnected(@Nullable Bundle bundle) {
...
-                public void onFailure(Call<ResponseBody> call, Throwable t) {
+            website.bloop.app.api.Player player = new website.bloop.app.api.Player(displayName, playerId);
 
-                }
-            });
+            // TODO: move this to another class?
+            mService.addPlayer(player)
+                    .subscribeOn(Schedulers.newThread())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribe(responseBody -> {
```
Comment (sammyers) [inline app/src/main/java/website/bloop/app/activities/PlayLoginActivity.java]: can we not have this

**Review by sammyers — APPROVED** [review]
Alright, looks good.

File: app/src/main/java/website/bloop/app/LocationModel.java (hunk 2 above)
Comment (wolfd) [reply app/src/main/java/website/bloop/app/LocationModel.java:8]: Still in progress, forgot to remove it

**Pull request merged**